test/config_tests.erl

```diff
@@ -572,4 +694,46 @@ getmsg(Pid) ->
 
 
 n_handlers() ->
-    length(gen_event:which_handlers(config_event)).
+    length(handlers()).
+
+handlers() ->
+    Handlers = gen_event:which_handlers(config_event),
+    [Pid || {config_listener, {?MODULE, Pid}} <- Handlers].
+
+%% copy/paste from test_util
```

Comment:
That last bit we can obviously punt on for now and then just work to fix over time.